emails/utils.py

```diff
@@ -132,7 +114,7 @@ def ses_send_raw_email(
     try:
         # Provide the contents of the email.
         emails_config = apps.get_app_config('emails')
-        emails_config.ses_client.send_raw_email(
+        ses_response = emails_config.ses_client.send_raw_email(
```

Comment:
Capturing and using the `ses_response` so we can use its ephemeral `MessageId` value to derive encryption keys.